### DISCUSSION TITLE
수정 전 자소서로 원서를 넣고 있는데 이후 수정한 원서로 바꾸면 회사측은 어떤원서를 보게 되나요?

### DISCUSSION THREAD

**Discussion by gksgpwls0306** [biancasuji/biancasuji_#6] [Q&A] [answered]
수정 전 자소서로 원서를 넣고 있는데 이후 수정한 원서로 바꾸면 회사측은 어떤원서를 보게 되나요? 열람하지 않았을 경우에!

**Comment by biancasuji** [comment, marked as answer]
입사할 때의 이력서를 보게 됩니다!! :) 
만약 접수 취소하고 다시 접수하면 그땐 수정된 이력서를 보게 되용!

**Reply by gksgpwls0306** [reply]
이거 정말 궁금했어요! 이력서를 수정하고 싶으면 취소하고 다시 접수해야하는군요!
